### ISSUE TITLE
bug: `ERROR: syntax error at or near "scores_avg"`

### ISSUE THREAD

**Issue by Manouchehri** [langfuse/langfuse#1045]
### Describe the bug

When I attempt to export a generation, I get this error message.

<img width="1106" alt="image" src="https://github.com/langfuse/langfuse/assets/7232674/f862f2f5-28ee-45e7-9f30-c56119e82a59">

```json
[
    {
        "error": {
            "json": {
                "message": "\nInvalid `prisma.$queryRaw()` invocation:\n\n\nRaw query failed. Code: `42601`. Message: `ERROR: syntax error at or near \"scores_avg\"`",
                "code": -32603,
                "data": {
                    "code": "INTERNAL_SERVER_ERROR",
                    "httpStatus": 500,
                    "path": "generations.export",
                    "zodError": null
                }
            }
        }
    }
]
```

### To reproduce

Not sure to be honest. I'm using PostgreSQL 15.4 on Google Cloud SQL.

**Comment by marcklingen** [comment]
Most likely fixed if you upgrade to 2.1.1: https://github.com/langfuse/langfuse/releases/tag/v2.1.1

Feel free to reopen issue if problem persists

**Comment by marcklingen** [comment]
🥷
